app/services/ecf/npq_profile_updater.rb

```diff
@@ -18,7 +18,7 @@ def call
     end
 
     def tsf_data_field_update
-      return if Rails.application.config.npq_separation[:ecf_api_disabled]
+      return if Feature.ecf_api_disabled?
```

Comment:
maybe we can add something similar to the call as well, even if it's called from a mass updater/rake task just incase?